Features/Language.Feature

```diff
@@ -0,0 +1,57 @@
+Feature: This Test suite Contains test scenarios for language module
+1.Create Language
+2.Edit language
+3.Delete Language
+
+Scenario Outline: A. Create a new Language record
+Given User log into Mars portal
+When User navigate to Language module
+And User create a new Language record <Language> <Level>
+Then the tooltip message should be "<Language> has been added to your languages"
+
+Examples: 
+| Language   | level           |
+| 'English   | 'Basic          |
+| 'Hindi'    | 'Conversational |
+| 'Japanese' | 'Native'        |
+| 'French'   | 'Fluent'        |
+
+Scenario: D. Create a language record without data
+Given User log into Mars portal
+And User create a new language record without data
+Then the tooltip message should be "please enter language and level"
+
+Scenario: C. Create a language record with duplicate data
+Given User log into Mars portal
+And User create a new language record 'English' 'Basic'
+Then the tooltip message should be "This language is already exist in your language list"
+
+Scenario Outline: B. Check the limit of the Language records
```

Comment:
Please increase your coverage around the features you  are testing here is an idea of what is missing 

• Negative testing with valid input
• Negative testing with invalid input
• Destructive testing

E.g of above scenarios
Happy path tests check basic functionality and the acceptance criteria of the API. We later extend positive tests to include optional parameters and extra functionality.

The next group of tests is negative testing where we expect the application to gracefully handle problem scenarios with both valid user input (for example, trying to add an existing username) and invalid user input (trying to add a username which is null).

Destructive testing is a deeper form of negative testing where we intentionally attempt to break the API to check its robustness (for example, sending a huge payload body in an attempt to overflow the system).